build.gradle

```diff
@@ -183,6 +183,16 @@ task rcaTest(type: Test) {
     testLogging.showStandardStreams = true
 }
 
+task rcaIt(type: Test) {
```

Comment:
Maybe we can make this a verb like other gradle tasks(build, assemble, etc) `runRcaIt`?